src/main/java/car/RacingResult.java

```diff
@@ -0,0 +1,50 @@
+package car;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RacingResult {
+
+    private int max = 0;
+    private List<Car> racingResult;
+    private List<Car> winnerCars;
+    private RacingResult() {}
```

Comment:
```suggestion
    private int max = 0;
    private List<Car> racingResult;
    private List<Car> winnerCars;
    
    private RacingResult() {
    }
```

공백라인을 활용하여 문맥을 나누고, 가독성을 높혀보세요.